docs/setup-guide/arktos-apiserver-partition.md

```diff
@@ -0,0 +1,154 @@
+
+# Setting up Partitioned Arktos Environment
+
+[![Go Report Card](https://goreportcard.com/badge/github.com/futurewei-cloud/arktos)](https://goreportcard.com/report/github.com/futurewei-cloud/arktos)
+[![LICENSE](https://img.shields.io/badge/license-apache%202.0-green)](https://github.com/futurewei-cloud/arktos/blob/master/LICENSE)
+
+## What this document is about
+
+This document serves as a manual to set up data partitioned Arktos cluster across multiple machines. It has the following features:
+
+1. Multiple API servers can be configured to serve in a single Kubernetes clusters. ETCD Data is partitioned across API servers.
+
+1. Multiple controller instances (currently we suppport replicaset controller and deployment controller) can be running in parralled in different processes (named as workload controller manager). Each of them handles a portion of the requests to controller 
+
+
+## How to set up partitioned Arktos environment
+
+### Prerequsite
+
+Currently we require setting up developer environment. See [dev environment setup](setup-dev-env.md)
+
+### Start ETCD and 1st Api Server, plus Kube CM, Workload Controller Manager, Kube Scheduler
+
+```
+$ cd $GOPATH/src/arktos
+$ ./hack/arktos-up.sh
+```
+Note: this will soon be changed.
+
+### Start apiserver on a worker host
+
+. Verify we can access etcd from the worker host
+```
+$ curl http://<1st master server ip>:2379
```

Comment:
It might not be a good way to verify etcd 
a simple way to check curl -L http://[ip]:2379/v2/keys

Reply:
Updated